index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+<meta charset="UTF-8">
+<meta http-equiv="X-UA-Compatible" content="IE=edge">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<title>Hello, Welome!</title>
+<link rel="stylesheet" href="style.css">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+</head>
+
+<body>
+    <header> 
+      <nav>
+         <p>Welcome!  <img src="/home/edwin/Portfolio-webpage/Images/Icon - Menu.png" class="image">  </p>
+            </nav>
+    <h1 class="box heading"> Hey there. I’m Edwin! </h1> 
+    <h2 class="box heading"> I’m a software developer :)</h2> 
+    <article class="section heading">
+       <p> I can help you build a product, feature or website. Have a look at some of my work and experience! If you like what you see and have a project you need coded, 
+        don’t hestiate to contact me.</p>
+     </article>
+   </header>
+     <main>
+     <section>
+        <p>         
+<a href="https://github.com/" class="image1"><img src="/home/edwin/Portfolio-webpage/Images/github.svg" alt="github"> </a> 
+<a href="https://ke.linkedin.com/" class="image1"><img src="/home/edwin/Portfolio-webpage/Images/Linkedin icon.svg" alt="LinkedIn"</a> 
+<a href="https://angel.co/" class="image1"><img src="/home/edwin/Portfolio-webpage/Images/angellist.svg" alt="Angellist"> </a>
+<a href="https://twitter.com/login" class="image1"><img src="/home/edwin/Portfolio-webpage/Images/twitter.svg" alt="twitter"></a> 
+<a href="https://medium.com/" class="image1"><img src="/home/edwin/Portfolio-webpage/Images/medium.svg" alt="medium"></a>     
```

Comment:
Done

Reply:
- [ ] Please implement the suggestions the previous code reviewers gave. Here also the same Kindly fix the path to the images. The image is not shown due to an incorrect path; it should be ./Images/github.svg instead of /home/edwin..../... same for all the images :+1: 

***Your design***
> ![Screenshot from 2022-12-09 20-19-16](https://user-images.githubusercontent.com/65129228/206757612-27f83175-17b2-425b-b9fc-a954578e5576.png)